scoring/score.py

```diff
@@ -1,9 +1,57 @@
+DATA = {
+    # Location -> (Owner, Points if owner matches)
+    'arena':              (None, 0),
+    'docking_area':       (None, 1),
+    'zone_0_raised_area': (0,    3),
+    'zone_1_raised_area': (1,    3),
+}
+
+
 class Scorer:
+    # Assumption: we will trust the *ordering* of the entries in the data we
+    # are given and *ignore* the actual 'time' values. We defer handling of
+    # equialent-time operations to the token controller.
```

Comment:
Typo: "equivalent"

Reply:
Huh, that must have been there in the SR2021 scorer too. 5a9e88b780a1bd83e9d9ac9086d083c9bbc0fe3d